utilities/cli/holohub.py

```diff
@@ -196,6 +202,14 @@ def _create_parser(self) -> argparse.ArgumentParser:
         run.add_argument(
             "--language", choices=["cpp", "python"], help="Specify language implementation"
         )
+        run.add_argument(
+            "--run_args", help="Additional arguments to pass to the application executable"
+        )
+        run.add_argument(
+            "--with",
```

Comment:
do you think this should be `build_with` as well @tbirdso? this option is eventually only being used by './holohub build`